libs/utils/android/system.py

```diff
@@ -23,17 +23,28 @@
 import pexpect as pe
 
 GET_FRAMESTATS_CMD = 'shell dumpsys gfxinfo {} > {}'
+SYSTRACE_EVENTS_DEFAULT = ['gfx', 'view', 'sched', 'freq', 'idle']
+SYSTRACE_BUFSIZE_DEFAULT = 10240
 
 class System(object):
     """
     Collection of Android related services
     """
 
     @staticmethod
-    def systrace_start(target, trace_file, time=None,
-                       events=['gfx', 'view', 'sched', 'freq', 'idle']):
+    def systrace_start(target, trace_file, time=None):
 
         log = logging.getLogger('System')
+        conf = target.conf['systrace']
```

Comment:
I've just realised that this `target` is totally misnamed, it's actually a `TestEnv`. Can we refactor that first? In fact we have a lot of asymmetry here between systrace and ftrace :cry: . I think we need to spend some time to move this whole shebang into devlib anyway.